From 23d6f7db622102a9f5af552e98ab700e302babd9 Mon Sep 17 00:00:00 2001
From: Wally Gleemer <Wally.Gleemer@windriver.com>
Date: Tue, 25 Mar 2008 16:21:34 -0700
Subject: [PATCH] cavium: relocate pci console proto

Put the prototype for the pci console fcn up at the top with
all the others.

Signed-off-by: Wally Gleemer <Wally.Gleemer@windriver.com>
---
 arch/mips/cavium-octeon/setup.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/mips/cavium-octeon/setup.c b/arch/mips/cavium-octeon/setup.c
index c01e3a4..85795cf 100644
--- a/arch/mips/cavium-octeon/setup.c
+++ b/arch/mips/cavium-octeon/setup.c
@@ -34,6 +34,7 @@
 #include "cvmx-bootmem.h"
 
 extern void octeon_user_io_init(void);
+extern void pci_console_init(const char *arg);
 static unsigned long CYCLES_PER_JIFFY;
 static int ECC_REPORT_SINGLE_BIT_ERRORS = 0;
 static unsigned long long MAX_MEMORY = 512ull << 20;
@@ -808,7 +809,6 @@ void prom_init(void)
 	int argc;
 	struct uart_port octeon_port;
 	int octeon_uart;
-	extern void pci_console_init(const char *arg);
 
 	octeon_hal_init();
 	octeon_check_cpu_bist();
-- 
1.5.5.1

